README.md

```diff
@@ -621,32 +651,67 @@ For the client to prove possession of their cryptographic materials, they must c
+Credential Response can be immediate or deferred. If the issuer can immediately issue the requested credential, it will
+return it in the `credential` field, with an HTTP 200 (OK) status code. 
+
+If the issuer cannot immediately issue a credential it returns a `transaction_id`, which is to be used in a subsequent call 
+in the [Deferred Credential Request](#deferred-credential-request). The HTTP status code MUST be 202 (Accepted).
 
 | Field            | Description                                                                                | Required | References                                                                                           | Comments                                      |
 | :--------------- | :----------------------------------------------------------------------------------------- | :------- | :--------------------------------------------------------------------------------------------------- | :-------------------------------------------- |
 | `credential`     | The credential in `jwt_vc_json` format                                                     | n        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-7.3-6.1) | If missing, `transaction_id` must be present  |
 | `transaction_id` | ID used for subsequent call to [Deferred Credential Request](#deferred-credential-request) | n        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-7.3-6.2) | If missing, then `credential` must be present |
 
-> [!WARNING]
-> TODO we need to define the c_nonce stuff for the deferred flow; right now `proof` is embedded under the Credential Request, but it's applicable against the deferred and batch credential requests
+#### Credential Error Response
+
+##### Authorization Errors
+
+If the Credential Request does not contain a valid Access Token, the response is an authorization error response, as per [RFC6750](https://www.rfc-editor.org/rfc/rfc6750.html#section-3).
+
+#### Credential Request Errors
+
+| Field               | Description                               | Required | References                                                                                                 | Comments |
+| :------------------ | :---------------------------------------- | :------- | :--------------------------------------------------------------------------------------------------------- | :------- |
+| `error`             | A string error code from below list       | y        | [OID4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-7.3.1.2-3.1.1) |          |
```

Comment:
Might be good to list them out below (as well as the link you've added) like you did in the _Token Error Response_ section above

Reply:
yep, done